suite/tests/client-interface/drx-scattergather-bbdup.dll.c

```diff
@@ -61,6 +61,7 @@ inscount(uint num_instrs)
     global_sg_count += num_instrs;
 }
 
+#if defined(X86)
```

Comment:
What about AArch64 equivalent tests -- are these state restore tests at points in the expansion where key registers are clobbered?  Should there be a TODO i#5036 here?

Reply:
Generally, please reply with a message before resolving so it's clear why it was resolved.

Reply:
> Should there be a TODO i#5036 here?
I have added TODOs to this file and drx-scattergather.dll.c

> Generally, please reply with a message before resolving so it's clear why it was resolved.

Sorry. I will do that from now on.